src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/tya/NotificationsIt.java

```diff
@@ -903,7 +898,7 @@ public void shouldSendSubscriptionUpdatedNotificationForSubscriptionUpdatedReque
         HttpServletResponse response = getResponse(getRequestWithAuthHeader(json));
 
         assertHttpStatus(response, HttpStatus.OK);
-        verify(notificationClient, times(1)).sendEmail(any(), any(), any(), any());
+        verify(notificationClient, times(2)).sendEmail(any(), any(), any(), any());
```

Comment:
Is there any way we can break this down to verify what types of emails are sent here so we can be sure it's sending the correct ones?